src/main/java/com/github/manolo8/darkbot/core/objects/Gui.java

```diff
@@ -45,8 +45,10 @@ public void update() {
         width = (int) Math.round(size.x);
         height = (int) Math.round(size.y);
         // Set pos relative to window size
-        x = (int) Math.round((MapManager.clientWidth - size.x) * 0.01 * pos.x);
-        y = (int) Math.round((MapManager.clientHeight - size.y) * 0.01 * pos.y);
+        //x = (int) Math.round((MapManager.clientWidth - size.x) * 0.01 * pos.x);
+        //y = (int) Math.round((MapManager.clientHeight - size.y) * 0.01 * pos.y);
+        x = x();
+        y = y();
```

Comment:
i wonder, could these simply be super.x, super.y (given that they're made protected in sprite object) ?
i think x and y in gui should be marked deprecated (with the intent that they should be obtained via getX/getY method instead